### ISSUE TITLE
Rule: Disallow {{action}} (both element and helper forms)

### ISSUE THREAD

**Issue by rwjblue** [ember-template-lint/ember-template-lint#835]
As part of the Octane efforts, we should add a new rule that disallows the usage of `{{action` in any location. This would supersede the usage of `no-element-action` in the `octane` preset.

Bad examples:

```hbs
<button {{action 'submit'}}>Submit</button>
```

```hbs
<FooBar @baz={{action 'submit'}} />
```

```hbs
{{yield (action 'foo')}}
```

```hbs
{{yield (action this.foo)}}
```

The rule should suggest the usage of `on` for the element modifier form, and `fn` for the helper form.

**Comment by lifeart** [comment]
I can take it, how we will call this rule?
`deprecations/lint-deprecated-action-helper`?
`no-action-helper`?

**Comment by bmish** [comment]
Note that we have two existing rules:
* [no-action-modifiers](https://github.com/ember-template-lint/ember-template-lint/blob/master/docs/rule/no-action-modifiers.md)
* [no-element-event-actions](https://github.com/ember-template-lint/ember-template-lint/blob/master/docs/rule/no-element-event-actions.md)

I would suggest calling the new rule simply `no-action` to indicate that it's more broad than the existing rules.

**Comment by rwjblue** [comment]
Agree with @bmish

**Comment by rwjblue** [comment]
Closed by #853
